src/actions/topicActions.js

```diff
@@ -1,19 +1,19 @@
-import { SubmissionError } from 'redux-form/immutable';
-
 import TopicApi from 'api/topicApi';
 import * as actions from './actionTypes';
 
 export const loadTopicsSuccess = topics => ({
   type: actions.LOAD_TOPICS_SUCCESS, topics
 });
 
+export const loadTopicsFailed = errors => ({
+  type: actions.LOAD_TOPICS_FAILED, errors
+});
+
 export const loadTopics = () =>
   async (dispatch) => {
-    try {
-      await TopicApi.getTopics().then((data) => {
-        dispatch(loadTopicsSuccess(data.topics));
-      });
-    } catch (err) {
-      throw new SubmissionError(err.errors);
-    }
+    await TopicApi.getTopics().then((data) => {
```

Comment:
You don't need to use `async & await` if you are using promises

Reply:
I changed this!